include/can/core/ids.hpp

```diff
@@ -163,4 +163,11 @@ enum class PipetteTipActionType {
     drop = 0x1,
 };
 
+/**
```

Comment:
This needs to be generated from python otherwise it's going to create conflicts later. As long as you have your branch open in the monorepo it should work to use the `update-headers` command. FYI it will use the data string from there, so if you want all this text in the comment, you should update the python `constants.py` file to reflect that.